src/pykeen/datasets/__init__.py

```diff
@@ -49,6 +51,8 @@
     'WN18',
     'WN18RR',
     'YAGO310',
+    'DRKG',
```

Comment:
This is an unrelated change, right?

Reply:
correct, but it was missing and this is a dataset related PR so I threw it in there (the readme wasn't showing it becuase I forgot to include it)